torch/nn/modules/module.py

```diff
@@ -24,6 +24,12 @@
 T = TypeVar('T', bound='Module')
 
 
+def _get_speacial_module_supported_device():
```

Comment:
(bikeshedding) should naming be more clear e.g. `_get_special_module_supported_device_types`?

Reply:
> (bikeshedding) should naming be more clear e.g. `_get_special_module_supported_device_types`?

yeah I have rename it and fix the typo.